### PR TITLE
Implemented backend api

### PR DESCRIPTION
**Pull request by JoshuaC930**
References #101 

Backend api can be accessed through http://[localhost:8001/healthy_idaho/](http://localhost:8001/healthy_idaho/)

### CODE REVIEW

**Comment by JoshuaC930** [comment]
[t:90]

File: backend/healthy_idaho/models.py
Comment (anthony-diep66) [inline backend/healthy_idaho/models.py:1]: good

File: backend/healthy_idaho/views.py
Comment (anthony-diep66) [inline backend/healthy_idaho/views.py:1]: good

File: backend/wait.sh
Comment (anthony-diep66) [inline backend/wait.sh:1]: remove, do UploadData.py after docker startup manually

File: docker/django/Dockerfile
Comment (anthony-diep66) [inline docker/django/Dockerfile:1]: remove

Comment (JoshuaC930) [reply docker/django/Dockerfile:1]: reverted changes, now UploadData.py is manual and now adds year column

**Pull request merged**